kubespawner/spawner.py

```diff
@@ -71,6 +73,22 @@ def pods(self):
         return self.resources
 
 
+class MultiNamespacePodReflector(MultiNamespaceResourceReflector):
```

Comment:
Since the Multi base class is really one line, I think this might be simpler with:

```python
class MultiNamespacePodReflector(PodReflector):
    omit_namespace = True
    list_method_name = 'list_pod_for_all_namespaces'
```

and avoid duplication of the PodReflector.

Same goes for the other cross-namespace reflectors

Reply:
I've gotten rid of the multinamespace classes entirely.